modules/vba_documents/README.yml

```diff
@@ -0,0 +1,342 @@
+openapi: '3.0.0'
```

Comment:
I like specifying directly in YAML vs. a DSL.

Having it centralized for this particular service is fine. For vets-api, I wonder if we can break it out into multiple .yml files - a top-level one and then per-component ones -  and have the docs controller merge the various paths and schemas into a single object for serving. Seems feasible, but it's a chunk of code to write.